### ISSUE TITLE
Geeting Error when we are using aws-chime-sdk-js 

### ISSUE THREAD

**Issue by koni-a** [aws/amazon-chime-sdk-js#244]
Hi Team,

Installed latest version of aws-chime-sdk-js 

when we are doing ng build getting below error 

ERROR in node_modules/amazon-chime-sdk-js/build/mediarecording/MediaRecording.d.ts:27:21 - error TS2304: Cannot find name 'RecordingState'.

27     recordingState: RecordingState;
                       ~~~~~~~~~~~~~~
node_modules/amazon-chime-sdk-js/build/mediarecording/MediaRecording.d.ts:1:23 - error TS2688: Cannot find type definition file for 'dom-mediacapture-record'.

1 /// <reference types="dom-mediacapture-record" />
                        ~~~~~~~~~~~~~~~~~~~~~~~
node_modules/amazon-chime-sdk-js/build/mediarecording/WebMMediaRecording.d.ts:11:23 - error TS2304: Cannot find name 'MediaRecorderOptions'.

11     readonly options: MediaRecorderOptions;
                         ~~~~~~~~~~~~~~~~~~~~
node_modules/amazon-chime-sdk-js/build/mediarecording/WebMMediaRecording.d.ts:20:27 - error TS2304: Cannot find name 'RecordingState'.

20     get recordingState(): RecordingState;
                             ~~~~~~~~~~~~~~
node_modules/amazon-chime-sdk-js/build/mediarecording/WebMMediaRecording.d.ts:1:23 - error TS2688: Cannot find type definition file for 'dom-mediacapture-record'.

1 /// <reference types="dom-mediacapture-record" />
                        ~~~~~~~~~~~~~~~~~~~~~~~

can you please help us to resolve issue

**Comment by andyh6969** [comment]
If you use the latest version from GIT (not npm) then it works again.

**Comment by simmkyu** [comment]
I confirmed that the bug happens in an app created by `@angular/cli` (tested using v9.1.1). In the Angular's TS setup, Chime SDK cannot import w3c MediaStream Recording interfaces.

To mitigate the issue, please install type definitions for w3c MediaStream Recording (https://www.npmjs.com/package/@types/dom-mediacapture-record) in your project. This package will provide types for `RecordingState` and `MediaRecorderOptions`.

```
// In your Angular project
npm install @types/dom-mediacapture-record
```

We will test and include this fix in Chime SDK.

**Comment by vidya-mohan** [comment]
Fix pushed to 1.3.0 version in NPM so resolving
